### PR TITLE
Us04 us87 from us86 automation slack

### LINKED ISSUE #65: Créer un canal slack à partir d'une idée

### LINKED ISSUE #85: Le nouveau canal Slack est lié à l'idée dont il est issu

### LINKED ISSUE #86: Le canal Slack créé est lié à l'utilisateur qui a créé l'idée

### LINKED ISSUE #87: Les adhérents à l'idée sont automatiquement intégré au canal Slack créé

### PR DESCRIPTION
**Pull request by benrcd**

### CODE REVIEW

File: src/Controller/SlackController.php [hunk 1]
```diff
@@ -3,12 +3,15 @@
 namespace App\Controller;
 
 use App\Entity\Idea;
+// use App\Repository\MembershipRepository;
```
Comment (Spolito24) [inline src/Controller/SlackController.php]: useless comment

**Review by Spolito24 — APPROVED** [review]

**Review by AurelOconnell — CHANGES_REQUESTED** [review]
Need to review Entity/User, there's some check failures on lines 60, 266, 271 and 280

**Review by AurelOconnell — COMMENTED** [review]
All good now! :)

File: src/Controller/SlackController.php [hunk 2]
```diff
@@ -21,27 +23,56 @@ public function index(): Response
...
-    public function createChannel(SlackService $slackService, Idea $ideaTitle, SluggerInterface $slugger): Response
-    {
-        $channelName = $ideaTitle->getTitle(); // Set the channel name based on idea name
+    #[Route('{id}/createchannel', name: 'create_channel')]
+    public function createChannel(
+        SlackService $slackService,
+        IdeaRepository $ideaRepository,
+        Idea $title,
+        SluggerInterface $slugger,
+        Idea $idea,
```
Comment (aldaitzwild) [inline src/Controller/SlackController.php:31]: Why you get twice the same Idea ?

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by AurelOconnell — APPROVED** [review]
ok, double idea removed

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**